platforms/chibios/converters/elite_c_to_stemcell/_pin_defs.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Mega Mind (@megamind4089)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Pindefs for v2.0.0
+// https://megamind4089.github.io/STeMCell/pinout/
+
+// Left side (front)
+#ifdef STEMCELL_UART_SWAP
+#    define D3 PAL_LINE(GPIOA, 3)
+#    define D2 PAL_LINE(GPIOA, 2)
+#else
+#    define D3 PAL_LINE(GPIOA, 2)
+#    define D2 PAL_LINE(GPIOA, 3)
+#endif
+//      GND
+//      GND
+#ifdef STEMCELL_I2C_SWAP
+#    define D1 PAL_LINE(GPIOB, 6)
+#    define D0 PAL_LINE(GPIOB, 7)
+#else
+#    define D1 PAL_LINE(GPIOB, 7)
+#    define D0 PAL_LINE(GPIOB, 6)
+#endif
+
+#define D4 PAL_LINE(GPIOA, 15)
+#define C6 PAL_LINE(GPIOB, 3)
+#define D7 PAL_LINE(GPIOB, 4)
+#define E6 PAL_LINE(GPIOB, 5)
+#define B4 PAL_LINE(GPIOB, 8)
+#define B5 PAL_LINE(GPIOB, 9)
+
+// Right side (front)
+//      RAW
```

Comment:
You have the same issue here. This is not RAW on Elite-C. The converter will not succeed on boards that use B0